src/level-ext.adoc

```diff
@@ -74,14 +74,14 @@ endif::[]
 11+| bit[0] - <<m_bit>> ({CAP_MODE_VALUE}-{cheri_cap_mode_name}, {INT_MODE_VALUE}-{cheri_int_mode_name})
 |Bits[4:3]| R | W | C | LM | EL | SL  | X | ASR | Mode^1^ |
 | 0-1   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |  ✔  | Mode^1^  | Execute + ASR (see <<infinite-cap>>)
-| 2-3   | ✔ |   | ✔ | ✔  | ✔  | ∞   | ✔ |     | Mode^1^  | Execute + Data & Cap RO
+| 2-3   | ✔ |   | ✔ | ✔  | ✔  | ∞^1^| ✔ |     | Mode^1^  | Execute + Data & Cap RO
 | 4-5   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |     | Mode^1^  | Execute + Data & Cap RW
-| 6-7   | ✔ | ✔ |   |    |    | N/A | ✔ |     | Mode^1^  | Execute + Data RW
+| 6-7   | ✔ | ✔ |   |    |    | ∞^1^| ✔ |     | Mode^1^  | Execute + Data RW
```

Comment:
This could be 0 or ∞, because it truly doesn't matter as we will never load a writable cap via this (bearing future transitive SL in mind). 0 might be slightly friendlier. One could imagine a structure made deeply not SL, with some function pointers in it. Those should be sealed, which is why I don't really think this will ever be bothersome. But if there were any unsealed ones, and they had SL reduced, I think it would be more expected that they lose C, not X, as a result (possibly, they really should be sealed, so who knows what is more useful there).

Given that it is probably not important, and the rule "lose X if SL not inf" is easier than something like: "lose X if SL if inf != W", I don't have strong feelings here. Maybe somebody else has thoughts here?

Reply:
I have changed it. And we've also changed the order of rules 9 and 10, and update (former) rule 9 to allow for SL=0 in this case.

Previously it was testing for SL (former rule 9) and then clearning SL (former rule 10) which seems the wrong way around

With the latest commit if you start with infinite permissions, just remove C then you stay in quadrant 1 and end on 6-7, and so retain the X permission.

Previously the X was lost which seemed a shame, as it could have been kept.